.code-samples.meilisearch.yaml

```diff
@@ -323,7 +360,7 @@ authorization_header_1: |-
 get_one_key_1: |-
   Key key = await client.GetKeyAsync("d0552b41536279a0ad88bd595327b96f01176a60c2243e906c52ac02375f9bc4");
 get_all_keys_1: |-
-  Result keyResult = await client.GetKeysAsync();
+  Result keyResult = await client.GetKeysAsync(new KeysQuery() { Limit = 3 });
```

Comment:
Don't you need a type in this Result? `Result<Key>` or something? You can also opt to use just `var keyResult`